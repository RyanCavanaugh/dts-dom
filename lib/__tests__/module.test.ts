import {ContextFlags, create, emit} from "../index"

describe("a module", () => {
    it("matches the snapshot", () => {
        const module = create.module('my-module')
        expect(emit(module)).toMatchSnapshot();
    });

    describe("with superfluous module rootFlag", () => {
        it("matches the snapshot", () => {
            const module = create.module('my-module')
            expect(emit(module, {rootFlags: ContextFlags.Module})).toMatchSnapshot();
        });
    });
});
