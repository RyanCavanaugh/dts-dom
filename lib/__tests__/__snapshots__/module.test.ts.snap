// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a module matches the snapshot 1`] = `
declare module 'my-module' {
}


`;

exports[`a module with superfluous module rootFlag matches the snapshot 1`] = `
declare module 'my-module' {
}


`;
